Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
There should be more explanations for this usage. Well, it works fine for now. However, a new feature which is separable menu view will be launched soon. It won't work with the feature in this way then.